### DISCUSSION TITLE
new translation file

### DISCUSSION THREAD

**Discussion by ModAlpha** [togisoft/jetpack_compose_country_code_picker#3] [General]
hello, 

could you please add this Arabic translation file?
[strings.xml.zip](https://github.com/togisoft/jetpack_compose_country_code_picker/files/8797217/strings.xml.zip)

**Comment by togisoft** [comment]
I will add it today. I'll update it with some edits. Thank you

**Reply by ModAlpha** [reply]
thanks

**Comment by ModAlpha** [comment]
@togisoft will you add the file today?

**Reply by togisoft** [reply]
yes I'm doing the final tests now.

**Reply by togisoft** [reply]
The update is live now. you can check

**Reply by ModAlpha** [reply]
many thanks

**Reply by togisoft** [reply]
you are welcome :) You can support the project by adding it to your favorites. :+1:

**Reply by ModAlpha** [reply]
yes for sure
